app/src/main/java/org/systers/mentorship/view/adapters/TasksAdapter.kt

```diff
@@ -52,18 +57,22 @@ class TasksAdapter(
         }
         else{
             itemView.cbTask.setOnClickListener {
-                val builder = AlertDialog.Builder(context)
-                builder.setTitle(context.getString(R.string.mark_task_title))
-                builder.setMessage(context.getString(R.string.mark_task_message))
-                builder.setPositiveButton(context.getString(R.string.yes)){dialog, which ->
-                    itemView.cbTask.isChecked=true
-                    markTask(item.id)
-                }
-                builder.setNegativeButton(context.getString(R.string.no)){dialog,which ->
-                    itemView.cbTask.isChecked=false
-                }
-                val dialog: AlertDialog = builder.create()
-                dialog.show()
+//                val builder = AlertDialog.Builder(context)
+//                builder.setTitle(context.getString(R.string.mark_task_title))
+//                builder.setMessage(context.getString(R.string.mark_task_message))
+//                builder.setPositiveButton(context.getString(R.string.yes)){dialog, which ->
+//                    itemView.cbTask.isChecked=true
```

Comment:
Sorry, missed this last time. Can you remove this as well?

Reply:
Got it. On it @epicadk